### PR TITLE
Added FLIF-GUI in the downloads section.

### PR DESCRIPTION
**Pull request by SylvainDebaudringhien**

### CODE REVIEW

**Comment by TheJaredWilcurt** [comment]
Hey, just tried it out, works as advertised. I think the text should be tweaked some to better match the existing style

```
<p><a href="https://filemill.net/#flif-gui">FLIF-GUI</a> - Drag &amp; drop, parallel batch processing, size statistics. (Windows).</p>
```

**Comment by SylvainDebaudringhien** [comment]
Hi. Thanks for trying it out... :)
Text updated.

File: index.html [hunk 1]
```diff
@@ -129,6 +129,7 @@ <h2 id="download">Download</h2>
 
     <p><a href="https://github.com/FLIF-hub/FLIF">FLIF source code</a></p>
     <p><a href="http://flif.info/UGUI_FLIF">UGUI: FLIF</a> - A GUI for FLIF</p>
+	<p><a href="https://filemill.net/#flif-gui">FLIF-GUI</a> - Drag &amp; drop, parallel batch processing, size statistics. (Windows).</p>
```
Comment (waldyrious) [inline index.html]:
```suggestion
    <p><a href="https://filemill.net/#flif-gui">FLIF-GUI</a> - Drag &amp; drop, parallel batch processing, size statistics. (Windows).</p>
```

**Review by TheJaredWilcurt — APPROVED** [review]
👍

**Pull request merged**